client/src/modules/journal/modals/trialBalanceMain.body.html

```diff
@@ -1,22 +1,9 @@
-<!-- exportation -->
-<div class="row text-right" style="margin: 5px;">
-  <div class="col-xs-12">
-    <button
-      id="export"
-      type="button"
-      class="btn btn-success"
-      ng-click="TrialBalanceMainBodyCtrl.exportGrid()">
-      <i class="fa fa-file-excel-o"></i> 
-      <span translate>FORM.BUTTONS.EXPORT</span>
-    </button>
-  </div>
-</div>
-
-<div class="row" style="margin: 5px;" ng-if="TrialBalanceMainBodyCtrl.showErrorButton">
-  <div class="col-xs-4">
+<!-- tools for data validation and export -->
+<div class="row " style="margin: 5px;">
```

Comment:
At some point we should go through these hard coded margins and see if there is a pattern that we can resolve in the less styles.